app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomView.kt

```diff
@@ -11,8 +12,9 @@ interface ChatRoomView : LoadingView, MessageView {
      *
      * @param dataSet The data set to show.
      * @param serverUrl The server URL.
+     * @param settings The server settings.
      */
-    fun showMessages(dataSet: List<Message>, serverUrl: String)
+    fun showMessages(dataSet: List<Message>, serverUrl: String, settings: Map<String, Value<Any>>?)
```

Comment:
Why not pass only the setting value, instead of all settings?

Reply:
I thought on passing all the settings because we may have other settings the app could use later to customize the messages.

Reply:
Yeah, if you would like to create one already, it would be better. It is nice because we can do all the conversions, formatting, etc on the background, freeing the main thread